src/components/date_picker/super_date_picker/quick_select_popover/refresh_interval.js

```diff
@@ -90,6 +90,23 @@ export class EuiRefreshInterval extends Component {
     );
   };
 
+  startRefresh = () => {
+    const { value, units } = this.state;
+    const isValid = value === '' || value <= 0;
```

Comment:
Got this logic backwards, the test here is actually determining if the value is _invalid_.
```suggestion
    const isValid = value !== '' || value > 0;
```